angular-ngrx-scss/src/app/repository/components/repository-details/repository-details.component.ts

```diff
@@ -1,8 +1,19 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   selector: 'app-repository-details',
   templateUrl: './repository-details.component.html',
   styleUrls: ['./repository-details.component.scss'],
 })
-export class RepositoryDetailsComponent {}
+export class RepositoryDetailsComponent implements OnInit {
+  owner = '';
+  repo = '';
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit(): void {
```

Comment:
suggestion: Would really prefer these values be pulled in from the store. This info already exists there. Creating selectors for them if they don't already exist would be wonderful.

Reply:
think IMHO, waiting for the store to retrieve the owner and the name is not worth it, coming from the org view when you already have the owner and using the info from the `paramMap` without waiting to resolve an API is better that waiting for the selector, because at a certain point when we move to this URL for ex `getsentry/sentry` we don't have the data.

Maybe we can bring up this talk to the shared channel :)